spinalcordtoolbox/scripts/sct_image.py

```diff
@@ -175,6 +182,12 @@ def main(argv=None):
     fname_in = arguments.i
     n_in = len(fname_in)
 
+    # If the user wants to fix sforms replace `Image` with the sform fixer
+    if arguments.set_sform_to_qform:
+        Image = read_and_fix_sform
+    else:
+        Image = spinalcordtoolbox.image.Image
```

Comment:
That way we wouldn't need this...

Reply:
Ah, I understand now why this was done. It was even explicitly mentioned in https://github.com/neuropoly/spinalcordtoolbox/pull/2858#discussion_r485985597.

I'm going to write a `TODO` to limit the scope of this PR.

Reply:
While patching `Image` like this does do a good job at limiting the scope of the PR, I think the benefit isn't worth leaving the script like this.

Reply:
Done in f9c6c08.